src/fidesops/models/privacy_request.py

```diff
@@ -186,10 +186,17 @@ def cache_drp_request_body(self, drp_request_body: DrpPrivacyRequestCreate) -> N
         drp_request_body_dict: Dict[str, Any] = dict(drp_request_body)
         for key, value in drp_request_body_dict.items():
             if value is not None:
-                cache.set_with_autoexpire(
-                    get_drp_request_body_cache_key(self.id, key),
-                    value,
-                )
+                # handle nested dict/objects
+                if not isinstance(value, (bytes, str, int, float)):
+                    cache.set_with_autoexpire(
+                        get_drp_request_body_cache_key(self.id, key),
+                        repr(value),
```

Comment:
I know this probably feels hacky but good choice in effort/reward trade-off if this is only for debug purposes as you say.

Looks like `value` can only ever be a key from [this schema](https://github.com/ethyca/fidesops/blob/579f028faa620b04c34e58187c1bb212f62bf535/src/fidesops/schemas/drp_privacy_request.py#L22), which should be just fine.

If we want to do more with this data though let's make sure we're breaking it down and storing it by key. We could write a util method to handle that automatically for Pydantic schemas for instance.